cards/en/sm10.json

```diff
@@ -11297,6 +11297,82 @@
       "large": "https://images.pokemontcg.io/sm10/195_hires.png"
     }
   },
+  {
+    "id": "sm10-195a",
+    "name": "Dedenne-GX",
+    "supertype": "Pokémon",
+    "subtypes": [
+      "Basic",
+      "GX"
+    ],
+    "hp": "160",
+    "types": [
+      "Lightning"
+    ],
+    "rules": [
+      "Pokémon-GX rule: When your Pokémon-GX is Knocked Out, your opponent takes 2 Prize cards."
+    ],
+    "abilities": [
+      {
+        "name": "Dedechange",
+        "text": "When you play this Pokémon from your hand onto your Bench during your turn, you may discard your hand and draw 6 cards. You can't use more than 1 Dedechange Ability each turn.",
+        "type": "Ability"
+      }
+    ],
+    "attacks": [
+      {
+        "name": "Static Shock",
+        "cost": [
+          "Lightning",
+          "Colorless"
+        ],
+        "convertedEnergyCost": 2,
+        "damage": "50",
+        "text": ""
+      },
+      {
+        "name": "Tingly Return-GX",
+        "cost": [
+          "Lightning",
+          "Colorless"
+        ],
+        "convertedEnergyCost": 2,
+        "damage": "50",
+        "text": "Your opponent's Active Pokémon is now Paralyzed. Put this Pokémon and all cards attached to it into your hand. (You can't use more than 1 GX attack in a game.)"
+      }
+    ],
+    "weaknesses": [
+      {
+        "type": "Fighting",
+        "value": "×2"
+      }
+    ],
+    "resistances": [
+      {
+        "type": "Metal",
+        "value": "-20"
+      }
+    ],
+    "retreatCost": [
+      "Colorless"
+    ],
+    "convertedRetreatCost": 1,
+    "number": "195",
+    "artist": "kanahei",
+    "rarity": "Rare Ultra",
+    "nationalPokedexNumbers": [
+      702
+    ],
+    "legalities": {
+      "unlimited": "Legal",
+      "standard": "Legal",
+      "expanded": "Legal"
+    },
+    "images": {
+      "small": "INSERT_NEW_IMAGE",
+      "large": "INSERT_NEW_IMAGE"
```

Comment:
We need to update this before merging, feel free to do so @adback03 since I honestly have no idea how to help here :(

Reply:
Can you update these with the appropriate urls? I will find the images, no worries. You can just follow the same image format as other cards:

https://images.pokemontcg.io/sm10/195a.png
https://images.pokemontcg.io/sm10/195a_hires.png

Reply:
I'm so blind sometimes 🤦 
Done! ✅